ogcserver/WMS.py

```diff
@@ -47,10 +48,12 @@ def __init__(self, configpath=None):
         self.styles = {}
         self.aggregatestyles = {}
         self.map_attributes = {}
+        self.map_scale = 1
         self.meta_styles = {}
         self.meta_layers = {}
         self.configpath = configpath
         self.latlonbb = None
+        log = logging.getLogger('ogcserver.wsgi')
```

Comment:
@pthorin `log` is not used right? can you please remove it and also the `logging` import?

Reply:
```suggestion
```